Blockzilla/TipManager.swift

```diff
@@ -78,6 +78,7 @@ class TipManager {
     lazy var shareTrackersTip = Tip(title: UIConstants.strings.shareTrackersTipTitle, identifier: TipKey.shareTrackersTip)
     
     func fetchTip() -> Tip? {
+        guard UserDefaults(suiteName: AppInfo.sharedContainerIdentifier)?.bool(forKey: SettingsToggle.showHomeScreenTips.rawValue) == false else { return shareTrackersTip }
```

Comment:
I'm not sure if we even show the share trackers tip if they've disabled tips. Maybe @sblatz knows?

Reply:
My thought was that the user would want to hide tips so that they can always see the tracker count but I'm interested to hear other opinions.

Reply:
@pdil confirmed this is how it should behave--once these nits are fixed I can merge it in! 😄